res/raw/webxdc_wrapper.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="color-scheme" content="dark light">
```

Comment:
this is for avoiding some flickering? probably okay as long as this affects the loader only and not the loaded iframe.

for the webxdc as such, dark mode cannot be forced from outside, cmp. https://github.com/deltachat/deltachat-android/pull/2208

Reply:
I guess this is to avoid the white screen with the FILL500 loading, I have same concerns as r10s wrt if this apply also to the contained iframe

Reply:
> as long as this affects the loader only and not the loaded iframe

That's right, it doesn't affect the inside of the `<iframe>`